x-pack/plugins/remote_clusters/__jest__/client_integration/remote_clusters_edit.test.js

```diff
@@ -60,11 +53,13 @@ describe.skip('Edit Remote cluster', () => {
    * the "create" remote cluster, we won't test it again but simply make sure that
    * the form component is indeed shared between the 2 app sections.
    */
-  test('should use the same Form component as the "<RemoteClusterEdit />" component', async () => {
+  test('should use the same Form component as the "<RemoteClusterAdd />" component', async () => {
     const { component: addRemoteClusterComponent } = setupRemoteClustersAdd();
 
-    await nextTick();
-    addRemoteClusterComponent.update();
+    await act(async () => {
+      await nextTick();
```

Comment:
Is it possible to also have a `waitFor()` here instead of the `nextTick` + `component.update()`?

Reply:
Good point. Updated this to use `waitFor()`.